modules/install/pages/deployment-considerations-lt-3nodes.adoc

```diff
@@ -32,29 +33,57 @@ In the event of a failure, running the cluster at this level ensures that the re
 
-* The REST API, see xref:rest-api:rest-set-up-services.adoc[Assigning Services].
+The arbiter node fulfills a similar role when there are two xref:learn:clusters-and-availability/groups.adoc[server groups]. 
+In case of network partitioning, the arbiter node can make sure that only one server group continues processing.
+To play this role, the arbiter node should be in a third server group to maximize the chance of it remaining available if one of the server groups fails. 
+Ideally, you should locate it in a separate rack when using physical hardware.
+When it runs in a container, cloud instance, or as a virtual machine, locate it on a separate host machine from the server groups.
 
-* Couchbase Web Console during node-addition and node-joining, see the demonstrated uses of checkboxes, in xref:manage:manage-nodes/add-node-and-rebalance#arbiter-node-addition[Add a Node and Rebalance] and in xref:manage:manage-nodes/join-cluster-and-rebalance#arbiter-node-join[Join a Cluster and Rebalance].
+To deploy an arbiter node, add or join a node to the cluster without assigning it any services. 
+For instructions on adding a node, see:
 
-* The CLI during node-addition, see xref:cli:cbcli/couchbase-cli-server-add[server-add].
+*  xref:rest-api:rest-set-up-services.adoc[Assigning Services] to deploy via the REST API.
+
+* xref:manage:manage-nodes/add-node-and-rebalance#arbiter-node-addition[Add a Node and Rebalance] and xref:manage:manage-nodes/join-cluster-and-rebalance#arbiter-node-join[Join a Cluster and Rebalance] to deploy via the Couchbase Server Web Console.
+
+* The xref:cli:cbcli/couchbase-cli-server-add[server-add] command to deploy via the command line interface.
+
+For more information about failover, see xref:learn:clusters-and-availability/hard-failover.adoc[Hard Failover], and in particular, xref:learn:clusters-and-availability/hard-failover.adoc#default-and-unsafe[Hard Failover in Default and Unsafe Modes].
+
+==== Arbiter Nodes and Licenses
+
+An arbiter node does not run any services other than the xref:learn:clusters-and-availability/cluster-manager.adoc[].
+This service is necessary to make the arbiter node part of the Couchbase Server cluster.
+Because it does not run other services, such as the xref:learn:services-and-indexes/services/data-service.adoc[], the arbiter node does not require an additional license to run. 
+It's essentially free aside from the cost of running the physical hardware, container, or cloud instance. 
+
+==== Sizing Recommendations for Arbiter Nodes
+
+Because the arbiter node runs only the Cluster Manager service, it often does not require the same resources as other nodes in the cluster.
+Your best option for choosing hardware or a cloud instance for an arbiter node is to start with the smallest instance you plan to or are already using in your cluster.
+Then monitor the arbiter node's CPU and RAM usage when your cluster experiences failovers and rebalanaces.
```

Comment:
It may be worth mentioning here that the node's CPU and RAM usage (particularly during rebalances) will increase as various things increase in the cluster, including nodes, buckets, and collections, and, as a result, there is no one-size fits all node.